tests/winding.rs

```diff
@@ -73,9 +76,9 @@ fn test<P, I>(poly_iter: I) where
 #[test]
 fn wind_plane() {
     test(generators::Plane::new()
-        .map(|p| p.map_vertex(|v| [v[0], v[1], 1f32])));
+        .map(|p| p.map_vertex(|mut v| {v.pos[2] = 1.; v})));
     test(generators::Plane::subdivide(3, 4)
-        .map(|p| p.map_vertex(|v| [v[0], v[1], 1f32])));
+        .map(|p| p.map_vertex(|mut v| {v.pos[2] = 1.; v})));
```

Comment:
The trait MapToVertices does this a little more elegantly.

https://docs.rs/genmesh/0.4.3/genmesh/trait.MapToVertices.html

```rust
.vertex(|mut v| {v.pos[2] = 1.; v}));
```